api/internal/core/entity/entity.go

```diff
@@ -83,6 +83,7 @@ type Route struct {
 	ServiceProtocol string                 `json:"service_protocol,omitempty"`
 	Labels          map[string]string      `json:"labels,omitempty"`
 	EnableWebsocket bool                   `json:"enable_websocket,omitempty"`
+	Status          uint8                  `json:"status"`
```

Comment:
We have type `Status`, so why use `uint8` here?

Reply:
`status` only expect 0 or 1, so define the type of `Status` is `uint8`

Reply:
fixed.